yacc.go

```diff
@@ -3470,22 +3469,20 @@ func $$Parse($$lex $$Lexer) int {
 
 func ($$rcvr *$$ParserImpl) Parse($$lex $$Lexer) int {
 	var $$n int
-	var $$lval $$SymType
```

Comment:
Ah, I see. I guess the escape analysis is not as sophisticated as I thought (the lexer is a stateless function). Thanks again.